examples/kafka/verify.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -e
+
+export NAME=kafka
+export PORT_PROXY="${KAFKA_PORT_PROXY:-11100}"
+export PORT_ADMIN="${KAFKA_PORT_ADMIN:-11101}"
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+kafka_client () {
+    docker-compose run --rm kafka-client "$@"
+}
+
+TOPIC="envoy-kafka-broker"
+
+MESSAGE="Welcome to Envoy and Kafka broker filter!"
+
+run_log "Create a Kafka topic"
+kafka_client kafka-topics --bootstrap-server proxy:10000 --create --topic $TOPIC
+
+run_log "Check the Kafka topic"
+kafka_client kafka-topics --bootstrap-server proxy:10000 --list | grep $TOPIC
+
+run_log "Send a message using the Kafka producer"
+kafka_client /bin/bash -c " \
+    echo $MESSAGE \
+    | kafka-console-producer --request-required-acks 1 --broker-list proxy:10000 --topic $TOPIC"
+
+run_log "Receive a message using the Kafka consumer"
+kafka_client kafka-console-consumer --bootstrap-server proxy:10000 --topic $TOPIC --from-beginning --max-messages 1 | grep "$MESSAGE"
+
+run_log "Check admin kafka_broker stats"
+EXPECTED_BROKER_STATS=(
+    # "kafka.kafka_broker.request.create_topics_request: 1"
```

Comment:
cc @adamkotwasinski this one doesnt seem to work as expected (returns 0) so is commented out for now